core/src/main/java/org/apache/metamodel/query/QueryConstants.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.metamodel.query;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class shall hold all constant configurable variables related to Query
+ * parsing
+ * 
+ * @author ashwinrayaprolu
+ *
+ */
+public final class QueryConstants {
+	/**
+	 * This field will hold start and end character for delimiter that
+	 * can be used
+	 */
+	public static final Map<Character, Character> delimiterMap = new HashMap<Character, Character>();
```

Comment:
I think having a Map internally as a means towards the goal is fine, let's just not expose it as publically accessible API.